docs/CI.md

```
@@ -1,20 +1,495 @@
+There are two noteworthy forms of feedback from the various linting automation,
+[code-scanning reports](https://docs.github.com/en/code-security/code-scanning), and
+[GitHub Anotated Messages](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-debug-message).
+All linting workflows failures result in alerts that can be reviewed with the relevant
+[PR via GitHub's UI](https://docs.github.com/en/code-security/code-scanning/managing-code-scanning-alerts/triaging-code-scanning-alerts-in-pull-requests#code-scanning-results-check-failures),
+albeit with different levels of details per linter workflow and target branch.
+
+#### Linting phases
+
+The generalized design of linter workflows in the Multicast Project CI/CD pipeline follows these
+same phases:
+  * **Initialization** - Bootstraps environment and any initial setup automatically
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)